tests/Configurator/DockerComposeConfiguratorTest.php

```diff
@@ -114,7 +115,7 @@ class DockerComposeConfiguratorTest extends TestCase
     /** @var DockerComposeConfigurator */
     private $configurator;
 
-    /** @var Package */
```

Comment:
Trying here to fix: https://github.com/symfony/flex/runs/5509940830?check_suite_focus=true